tahoma_api/client.py

```diff
@@ -141,16 +117,41 @@ async def get_current_executions(self) -> List[Any]:
 
         return response
 
+    async def send_command(
+        self,
+        device_url: str,
+        command: Command,
+        label: Optional[str] = "python-tahoma-api",
+    ) -> str:
+        """ Send a command """
+        return await self.send_commands(device_url, [command], label)
+
+    async def send_commands(
+        self,
+        device_url: str,
+        commands: List[Command],
+        label: Optional[str] = "python-tahoma-api",
+    ) -> str:
+        """ Send several commands in one call """
+        payload = {
+            "label": label,
+            "actions": [{"deviceURL": device_url, "commands": commands}],
+        }
+        response = await self.__post("exec/apply", payload)
+        return response["execId"]
```

Comment:
On purpose I've not reintroduce the supported_modes. If you know there are used, I can add it.

Reply:
Not at the moment, but in the end, the highPriority and internal could be useful.

Reply:
So, do you want I add them again?